content/posts/4-music-mot-nha-da-lab.md

```diff
@@ -0,0 +1,583 @@
+    - I go here and there performing only to go home at night with nothing
+- {{< tooltip "Thank" >}}Cảm ơn{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "because" >}}vì{{< /tooltip >}}
+  {{< tooltip "indicates plural" >}}những{{< /tooltip >}}
+  {{< tooltip "dates" >}}tháng ngày{{< /tooltip >}}
+  {{< tooltip "last" >}}qua{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "indicates a past tense" >}}đã{{< /tooltip >}}
+  {{< tooltip "to come" >}}đến{{< /tooltip >}}
+  {{< tooltip "side" >}}bên{{< /tooltip >}}
+  {{< tooltip "my (a man), because the man speaks" >}}anh{{< /tooltip >}}
+    - Thank you for all these times you've always been by my side
+- {{< tooltip "Thank" >}}Cảm ơn{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "indicates a past tense" >}}đã{{< /tooltip >}}
+  {{< tooltip "is" >}}là{{< /tooltip >}}
+  {{< tooltip "rear" >}}hậu phương{{< /tooltip >}}
+  {{< tooltip "for" >}}cho{{< /tooltip >}}
+  {{< tooltip "dreams, wishes, hopes" >}}ước mơ{{< /tooltip >}}
+  {{< tooltip "within" >}}trong{{< /tooltip >}}
+  {{< tooltip "my (a man), because the man speaks" >}}anh{{< /tooltip >}}
+    - Thank you for having been supporting my dreams
+- {{< tooltip "Sorry" >}}Xin lỗi{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "because" >}}vì{{< /tooltip >}}
+  {{< tooltip "indicates plural" >}}những{{< /tooltip >}}
+  {{< tooltip "evening" >}}tối{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "awake" >}}thức{{< /tooltip >}}
+  {{< tooltip "late" >}}muộn{{< /tooltip >}}
+  {{< tooltip "to refrain" >}}nhịn{{< /tooltip >}}
+  {{< tooltip "hungry" >}}đói{{< /tooltip >}}
+  {{< tooltip "to wait" >}}chờ{{< /tooltip >}}
+  {{< tooltip "a rice meal (\"to eat\" in this context)" >}}cơm{{< /tooltip >}}
+    - I'm sorry for the nights you've stayed up late and hungry to wait for me
+- {{< tooltip "I (a man), because the man speaks" >}}anh{{< /tooltip >}}
+  {{< tooltip "very" >}}rất{{< /tooltip >}}
+  {{< tooltip "to love" >}}yêu{{< /tooltip >}}
+  {{< tooltip "you (a woman), because a man speaks" >}}em{{< /tooltip >}}
+  {{< tooltip "and" >}}và{{< /tooltip >}}
+  {{< tooltip "to remain" >}}còn{{< /tooltip >}}
+  {{< tooltip "song" >}}bài{{< /tooltip >}}
+  {{< tooltip "this" >}}này{{< /tooltip >}}
+  {{< tooltip "I (a man), because the man speaks" >}}anh{{< /tooltip >}}
+  {{< tooltip "\"would like\"" >}}xin{{< /tooltip >}}
+  {{< tooltip "to write" >}}viết{{< /tooltip >}}
+  {{< tooltip "a gift" >}}tặng{{< /tooltip >}}
+  {{< tooltip "two" >}}hai{{< /tooltip >}}
+  {{< tooltip "mother" >}}mẹ{{< /tooltip >}}
+  {{< tooltip "child" >}}con{{< /tooltip >}}
+    - I love you so much and through this post
+    - I would like to write for you and our child
```

Comment:
> Ϲảm ơn em đã là hậu phương cho ước mơ trong anh
> Xin lỗi em vì những tối em thức muộn nhịn đói chờ cơm
> Anh rất yêu em và còn bài này anh xin viết tặng hai mẹ con

Does `"hai mẹ con"` mean "the mother and the child" in this context?

If so, can we use `"I would like to write for you and our child"`?

Reply:
@thaithimyduyen